docs/apis-tools/testing/configuration.md

```diff
@@ -0,0 +1,194 @@
+{label: 'Java client', value: 'java-client' }
+]
+}>
+
+<TabItem value='spring-sdk'>
+
+In your `application.yml` (or `application.properties`):
+
+```yaml
+io:
+  camunda:
+    process:
+      test:
+        # Change the version of the Camunda Docker image
+        camunda-docker-image-version: 8.8.0
+        # Change the Camunda Docker image
+        camunda-docker-image-name: camunda/camunda
+        # Set additional Camunda environment variables
+        camunda-env-vars:
+          env_1: value_1
+        # Expose additional Camunda ports
+        camunda-exposed-ports:
+          - 4567
+        # Enable Connectors
+        connectors-enabled: true
+        # Change the Connectors Docker image
+        connectors-docker-image-name: camunda/connectors
+        # Change version of the Connectors Docker image
+        connectors-docker-image-version: 8.8.0
+        # Set additional Connectors environment variables
+        connectors-env-vars:
+          env_1: value_1
+        # Set Connectors secrets
+        connectors-secrets:
+          secret_1: value_1
+```
+
+</TabItem>
+
+<TabItem value='java-client'>
+
+In your `/camunda-container-runtime.properties` file:
+
+```properties
+camunda.dockerImageVersion=8.8.0
+```
+
+For more configuration options, you can register the JUnit extension manually and use the fluent builder:
+
+```java
+package com.example;
+
+import io.camunda.process.test.api.CamundaProcessTestExtension;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+// No annotation: @CamundaProcessTest
+public class MyProcessTest {
+
+    @RegisterExtension
+    private static final CamundaProcessTestExtension EXTENSION =
+        new CamundaProcessTestExtension()
+            // Change the version of the Camunda Docker image
+            .withCamundaDockerImageVersion("8.8.0")
+            // Change the Camunda Docker image
+            .withCamundaDockerImageName("camunda/camunda")
+            // Set additional Camunda environment variables
+            .withCamundaEnv("env_1", "value_1")
+            // Expose additional Camunda ports
+            .withCamundaExposedPort(4567)
+            // Enable Connectors
+            .withConnectorsEnabled(true)
+            // Change the Connectors Docker image
+            .withConnectorsDockerImageName("camunda/connectors")
+            // Change version of the Connectors Docker image
+            .withConnectorsDockerImageVersion("8.8.0")
+            // Set additional Connectors environment variables
+            .withConnectorsEnv("env_1", "value_1")
+            // Set Connectors secrets
+            .withConnectorsSecret("secret_1", "value_1");
+}
+```
+
+</TabItem>
+
+</Tabs>
+
+## Remote runtime
+
+Instead of using the managed [Testcontainers runtime](#testcontainers-runtime), you can configure CPT to connect to a remote runtime, for example, to [Camunda 8 Run](/docs/self-managed/setup/deploy/local/c8run.md) on your machine.
```

Comment:
```suggestion
Instead of using the managed [Testcontainers runtime](#testcontainers-runtime), you can configure CPT to connect to a remote runtime, for example, to [Camunda 8 Run](/self-managed/setup/deploy/local/c8run.md) on your machine.
```